app/src/components/organisms/SurveyClassQuestion.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import Radio from '@mui/material/Radio';
+import RadioGroup from '@mui/material/RadioGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import FormControl from '@mui/material/FormControl';
+import FormLabel from '@mui/material/FormLabel';
+import Stack from '@mui/material/Stack';
+import { willingRadio, ability, willing } from '../shared/constants/surveyForm.constants';
+
+function SurveyClassQuestion(props: {
+  name: string;
+  fieldChanged: (fieldId: string, type: string, value: string) => void;
+}) {
+  const [disabled, setDisabled] = useState(true);
+  const [checked, setChecked] = useState({ willing: false, notWilling: false, veryWilling: false });
+
+  const handleDisabling = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (event.target.value === ability.cannot) {
+      setDisabled(event.target.checked);
+
+      setChecked(() => {
+        return willingRadio;
+      });
+    } else {
+      setDisabled(!event.target.checked);
+      setChecked(() => {
+        return {
+          ...willingRadio,
+          notWilling: true
+        };
+      });
+    }
+    props.fieldChanged(props.name, 'ability', event.target.value);
+  };
+
+  const changeRadio = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setChecked(() => {
+      return {
+        ...willingRadio,
+        [event.target.value]: true
+      };
+    });
+
+    props.fieldChanged(props.name, 'willing', (event.target as HTMLInputElement).value);
+  };
+
+  return (

```

Comment:
I think we are missing the element of those courses where the professor _has_ to have a specific qualification like a PENG. I think the "can't teach" radio implies this, but we should be more specific imo. I think the backend _should_ store this information but I dont think they are right now so we'll need to get it. For now we could add an asterisk to certain courses and show in the UI that an asterisk means additional qualifications are necessary.

Reply:
Added a additionalQualifications prop to surveyClassQuestion that just adds required to the radio. This doesn't actually do anything other than display an asterisk